src/module/types.rs

```diff
@@ -23,6 +23,17 @@ impl ModuleTypes {
         &mut self.arena[id]
     }
 
+    /// Get a type ID by its name.
```

Comment:
Perhaps worth mentioning that this is only intended for in-memory modifications and by default this always returns `None` for any newly-deserialized module?